critical/modules/secrets/distributed/main.tf

```diff
@@ -1,5 +1,5 @@
 module "secrets" {
-  source = "../secret"
+  source = "github.com/wellcomecollection/terraform-aws-secrets.git?ref=8bf3759af9e1a19731f28784953dd0a53a2f276f"
```

Comment:
This depends on https://github.com/wellcomecollection/terraform-aws-secrets/pull/4

Reply:
Reviewed this - we can update this.

Reply:
Done.